spec/unit/filter_spec.rb

```diff
@@ -338,7 +338,7 @@ def expect_file_scope_filter_change(scope, filter, before, after)
             "/usr/bin/replaced_by_link"
           ]
         )
-      }.to raise_error(Machinery::Errors::WrongFilterPath)
+      }.to raise_error(Machinery::Errors::WrongFilterPath, /Error: Check if the path/)
```

Comment:
I would add the complete line, or at least with the path, then the issue with the backslashes would have been shown by the test.